backend/src/api/transaction-utils.ts

```diff
@@ -116,7 +116,10 @@ class TransactionUtils {
   public extendMempoolTransaction(transaction: IEsploraApi.Transaction): MempoolTransactionExtended {
     const vsize = Math.ceil(transaction.weight / 4);
     const fractionalVsize = (transaction.weight / 4);
-    const sigops = !Common.isLiquid() ? this.countSigops(transaction) : 0;
+    let sigops = transaction.sigops;
```

Comment:
Note: Currently, electrs will return sigops for Liquid as well, might want to add the check for Liquid before this.